File: OneDrive/Documentos/Store_Herencia/Store_Herencia/src/store/herencia/Beauty.java
package store.herencia;
public class Beauty extends Product {
    private String type;
    public Beauty (String name, double price, String importDate, int stock, String type){
        super(name, price, importDate, stock);
        this.type = type;
    }
    @Override
    public String toString() {
        return super.getData() + String.format(", Type: %s", type);
    }
}

File: PycharmProjects/Store_Herencia/py/Beauty.py
from Product import Product

class Beauty(Product):
    def __init__(self, name, price, import_date, stock, type):
        super().__init__(name, price, import_date, stock)
        self.type = type

    def __str__(self):
        return super().get_data() + f"type: {self.type}"
